code/web/src/modules/user/api/actions.js

```diff
@@ -1,145 +1,175 @@
 // Imports
-import axios from 'axios'
-import { query, mutation } from 'gql-query-builder'
-import cookie from 'js-cookie'
+import axios from "axios";
+import { query, mutation } from "gql-query-builder";
+import cookie from "js-cookie";
 
 // App Imports
-import { routeApi } from '../../../setup/routes'
+import { routeApi } from "../../../setup/routes";
 
 // Actions Types
-export const LOGIN_REQUEST = 'AUTH/LOGIN_REQUEST'
-export const LOGIN_RESPONSE = 'AUTH/LOGIN_RESPONSE'
-export const SET_USER = 'AUTH/SET_USER'
-export const LOGOUT = 'AUTH/LOGOUT'
-export const CHANGE_EDIT_MODE = 'AUTH/CHANGE_EDIT_MODE'
-export const SAVE_PROFILE = 'AUTH/SAVE_PROFILE'
+export const LOGIN_REQUEST = "AUTH/LOGIN_REQUEST";
+export const LOGIN_RESPONSE = "AUTH/LOGIN_RESPONSE";
+export const SET_USER = "AUTH/SET_USER";
+export const LOGOUT = "AUTH/LOGOUT";
+export const CHANGE_EDIT_MODE = "AUTH/CHANGE_EDIT_MODE";
+export const SAVE_PROFILE = "AUTH/SAVE_PROFILE";
```

Comment:
Is there a way to prevent your linter from changing all of these quotes? I think it looks more readable with single quotes, but ultimately it's just a preference.

Reply:
Yeah sorry, I will be more cautious about pressing my linter button, as I too am in the single quotes party.